### PR TITLE
Upgrade prod deps, adopt new pickedpoint schema, support Python 3.9, and update Poetry

### PR DESCRIPTION
**Pull request by jlevin**

### CODE REVIEW

File: entente/landmarks/landmarker.py [hunk 1]
```diff
@@ -81,9 +81,10 @@ def transfer_landmarks_onto(self, target):
...
-                apply_surface_regressor(self._regressor, target.v),
-            )
-        )
+        return [
+            {
+                "name": point["name"],
+                "point": apply_surface_regressor(self._regressor, target.v)[i].tolist(),
+            }
+            for (i, point) in enumerate(self.landmarks)
+        ]
```
Comment (jlevin) [inline entente/landmarks/landmarker.py:90]: This seems not great.  But I couldn't figure out how to zip it into a list of dicts.

Comment (paulmelnikow) [reply entente/landmarks/landmarker.py:90]: Listcomp is the right way to go, but restore the `zip()` as it was and iterate over that. It gives you tuples `(name, point)`.

File: entente/test_cli.py [hunk 2]
```diff
@@ -28,12 +28,9 @@ def test_transfer_landmarks_cli(tmp_path):
         assert result.exit_code == 0
 
         transferred = load_landmarks("target.json")
+
         np.testing.assert_array_equal(
```
Comment (paulmelnikow) [inline entente/test_cli.py:32]: These aren't numpy arrays, are they? If they are identical dicts that's fine, then use `assert transferred == expected_landmarks`.

Comment (jlevin) [reply entente/test_cli.py:32]: They're each a list of dicts (ie: the new format).  So I guess that still works.

File: entente/landmarks/landmarker.py [hunk 3]
```diff
@@ -55,6 +55,10 @@ def load(cls, source_mesh_path, landmark_path):
     def _regressor(self):
         from ..surface_regressor import surface_regressor_for
 
+        # import pdb
+
+        # pdb.set_trace()
+
```
Comment (paulmelnikow) [inline entente/landmarks/landmarker.py:61]:
```suggestion
```

**Review by paulmelnikow — APPROVED** [review]

**Review by paulmelnikow — COMMENTED** [review]
Ah, looks like there's a bit left to do here

**Review by paulmelnikow — CHANGES_REQUESTED** [review]
Ah, looks like there's a bit left to do here

File: pyproject.toml [hunk 4]
```diff
@@ -20,20 +20,19 @@ classifiers = [
...
 click = {version = ">7.0,<9.0", optional = true}
-lacecore = {version = ">=2.3.0,<=3.0.0", extras = ["obj"]}
-meshlab-pickedpoints = {version = ">=2.0.0,<3", optional = true}
-numpy = "<1.19.0"
+lacecore = {version= ">=3.0.0a0", extras = ["obj"]}
+meshlab-pickedpoints = {version = ">=3.0.0", optional = true}
+numpy = "<1.19.3"
 ounce = ">=1.1.1,<2.0"
-polliwog = ">=2.0.0,<=3.0.0"
+polliwog = ">=3.0.0a0"
```
Comment (paulmelnikow) [inline pyproject.toml:30]:
```suggestion
polliwog = ">=3.0.0a0,<4"
```

File: pyproject.toml [hunk 5]
```diff
@@ -20,20 +20,19 @@ classifiers = [
...
 proximity = {version = ">=1.1.0,<2", optional = true}
 PyYAML = {version = ">=5.1", optional = true}
 scipy = {version = "*", optional = true}
 tqdm = "*"
-tri-again = {version = ">=1.1.0,<2.0", optional = true}
+tri-again = {version = ">=2.0.0a0", optional = true}
```
Comment (paulmelnikow) [inline pyproject.toml:35]:
```suggestion
tri-again = {version = ">=2.0.0a0,<3", optional = true}
```

File: pyproject.toml [hunk 6]
```diff
@@ -20,20 +20,19 @@ classifiers = [
...
 [tool.poetry.dependencies]
-# tri-again doesn't work in Python 3.9.
-python = ">=3.7,<3.9"
+python = ">=3.7,<4"
 cached_property = "*"
 click = {version = ">7.0,<9.0", optional = true}
-lacecore = {version = ">=2.3.0,<=3.0.0", extras = ["obj"]}
-meshlab-pickedpoints = {version = ">=2.0.0,<3", optional = true}
-numpy = "<1.19.0"
+lacecore = {version= ">=3.0.0a0", extras = ["obj"]}
```
Comment (paulmelnikow) [inline pyproject.toml:26]:
```suggestion
lacecore = {version= ">=3.0.0a0,<4", extras = ["obj"]}
```

File: pyproject.toml [hunk 7]
```diff
@@ -20,20 +20,19 @@ classifiers = [
...
+meshlab-pickedpoints = {version = ">=3.0.0", optional = true}
```
Comment (paulmelnikow) [inline pyproject.toml:27]:
```suggestion
meshlab-pickedpoints = {version = ">=3.0.0,<4", optional = true}
```

**Review by paulmelnikow — COMMENTED** [review]
Trying to make these a little safer

**Review by paulmelnikow — APPROVED** [review]

**Pull request merged**